contrib/opencensus-ext-stackdriver/opencensus/ext/stackdriver/stats_exporter/__init__.py

```diff
@@ -110,18 +130,14 @@ def default_monitoring_labels(self):
         return self._default_monitoring_labels
 
 
-class StackdriverStatsExporter(base_exporter.StatsExporter):
+class StackdriverStatsExporter(object):
     """Stats exporter for the Stackdriver Monitoring backend."""
 
-    def __init__(self,
-                 options=Options(),
-                 client=None,
-                 default_labels={},
-                 transport=async_.AsyncTransport):
+    def __init__(self, options=Options(), client=None):
```

Comment:
If people try something like `xyz = StackdriverStatsExporter()` and `xyz.options.project_id = 'foobar'`, this would overwrite the default argument. Do we want to prevent this case?

Reply:
That seems fine to me, any reason to prevent it?

Reply:
For example:
x = StackdriverStatsExporter()
x.options.project_id = 'foobar'
y = StackdriverStatsExporter()
# here y.options.project_id was polluted with 'foobar'

Reply:
Oh I see, yeah this shouldn't be mutable.

Reply:
Fixed in 0dc919f.